tests/examples/test_mass_delete_old_medias.py

```diff
@@ -0,0 +1,237 @@
+                'speaker_email': 'error@example.com',
+            },
+            {
+                'oid': 'four_years_ago',
+                'title': 'Four Years Ago',
+                'add_date': FOUR_YEARS_AGO.strftime('%Y-%m-%d 20:00:00'),
+                'categories': '',
+                'storage_used': 30 * 1024 ** 3,  # 30 GB
+                'views_last_year': 75,
+                'views_last_month': 1,
+                'speaker_email': 'john.doe@example.com',
+            },
+            {
+                'oid': 'five_years_ago',
+                'title': 'Five Years Ago',
+                'add_date': FIVE_YEARS_AGO.strftime('%Y-%m-%d 20:00:00'),
+                'categories': '',
+                'storage_used': 30 * 1024 ** 3,  # 30 GB
+                'views_last_year': 75,
+                'views_last_month': 1,
+                'speaker_email': 'john.doe@example.com | jane.doe@example.com',
+            },
+        ],
+        'lives': [
+            {
+                'oid': 'live_three_years_ago',
+                'title': 'Live: Three Years Ago',
+                'add_date': THREE_YEARS_AGO.strftime('%Y-%m-%d 20:00:00'),
+                'categories': '',
+                'storage_used': 30 * 1024 ** 3,  # 30 GB
+                'views_last_year': 75,
+                'views_last_month': 1,
+                'speaker_email': 'john.doe@example.com | june.doe@example.com',
+            },
+        ],
+    }
+
+
+@pytest.fixture()
+def api_client(catalog):
+    def mock_api_call(url, **kwargs):
+        if url == 'catalog/bulk_delete/':
+            return {
+                'statuses': {oid: {'status': 200} for oid in kwargs['data']['oids']}
+            }
+        elif url == 'catalog/get-all/':
+            return catalog
+
+    from ms_client.client import MediaServerClient
+
+    client = MediaServerClient()
+    client.api = mock.MagicMock(side_effect=mock_api_call)
+    with mock.patch('examples.mass_delete_old_medias.MediaServerClient', return_value=client):
+        yield client
+
+
+Message = namedtuple('Message', ['sender', 'recipient', 'message'])
+
+
+class MockSMTP:
+    def __init__(self):
+        self._logged_in = False
+        self.mailbox: list[Message] = []
+
+    def __enter__(self):
+        return self
+
+    def __exit__(self, _exc_type, _exc_val, _exc_tb):
+        pass
+
+    def login(self, sender, password):
+        assert sender == 'sender@example.com'
+        assert password == 's3cr3t'
+        self._logged_in = True
+
+    def sendmail(self, sender, recipient, message):
+        assert self._logged_in
+        if recipient == 'error@example.com':
+            raise smtplib.SMTPRecipientsRefused({'error@example.com': (550, b'User unknown')})
+        self.mailbox.append(Message(sender, recipient, message))
+
+    def has_mail(self, sender_address: str, recipient: str, oids: list[str]):
+        for mail in self.mailbox:
+            if (
+                mail.sender == sender_address
```

Comment:
What is the purpose of the sender filter ?
All emails should originate from `'sender@example.com'`, so an `assert` check on each email should be enough.

Reply:
`All emails should originate from 'sender@example.com'`

If the code works as expected, yes. Which is why it's reflected in the tests.